dist/pythonlibs/testrunner/spawn.py

```diff
@@ -39,7 +39,13 @@
 # By default never reset after the terminal is open unless explicitly requested
 # through an environment variable.
 TESTRUNNER_RESET_AFTER_TERM = int(os.environ.get('TESTRUNNER_RESET_AFTER_TERM')
-                                  or '0')
+                                  or 0)
```

Comment:
Why this change?

Reply:
I admit it is a unrelated change, but all other lines (`MAKE_TERM_STARTED_DELAY`, `MAKE_RESET_DELAY`, ...) in this file give a number as default and not a string with the number, so I adjusted it to the given majority within the file. I don't think its worthy of a own commit/PR.